libraries/eosiolib/core/eosio/datastream.hpp

```diff
@@ -900,7 +893,7 @@ datastream<Stream>& operator>>( datastream<Stream>& ds, std::tuple<Args...>& t )
  */
 template<typename DataStream, typename T, std::enable_if_t<std::is_class<T>::value && _datastream_detail::is_datastream<DataStream>::value>* = nullptr>
 DataStream& operator<<( DataStream& ds, const T& v ) {
-   boost::pfr::for_each_field(v, [&](const auto& field) {
+   bluegrass::meta::for_each_field(v, [&](const auto& field) {
```

Comment:
boost::pfr is the only one that I would say that we will still want to keep around. This works because all of the structs in the tests are manually reflected.  

But, PFR is used for structs that are not reflected (precise flat reflection).  So, we can still use that one small lib (it's also self contained https://github.com/boostorg/pfr), I had a couple of different solutions away from PFR more than a year ago which we will more than likely dust off and use in the next release to remove PFR.

Reply:
bluegrass::meta::for_each_field is a copy of boost::pfr that I did. If we keep using boost (i.e. include it in eosiolib includes) then I face again antler-run issue with mixing boost versions

Reply:
Understood.